spec/support/features/session_helper.rb

```diff
@@ -162,7 +162,7 @@ def acknowledge_and_confirm_recovery_code
       click_on button_text, class: 'recovery-code-continue'
 
       code_words.size.times do |index|
-        fill_in "recovery-#{index}", with: code_words[index].downcase
+        fill_in "recovery-#{index}", with: code_words[index].downcase + 'xxxx'
```

Comment:
@monfresh no server-side validations on the recovery code, other than that it matches what we have on file. We don't check the minimum length of a user password when we are validating it -- only that it matches. I'd argue the same logic applies to the recovery code.